docs/nextflow.md

```diff
@@ -18,3 +19,87 @@ cat nextflow.install.bash | bash -i 2>&1 | tee nextflow.install.log
 # run test, see https://www.nextflow.io/index.html#GetStarted
 ./nextflow run hello
 ```
+
+Configure the environment you're using if you haven't done so yet.
+Don't forget to add `NXF_HOME`, patch `PATH` and export them.
+```
+# fix env variables, i.e.:
+export NXF_HOME=$(pwd)/dot.nextflow
+export PATH=$(pwd):$PATH
+```
+
+If you wish, you can set `NXF_WORK` env to be used by `nextflow`.
+```
+export NXF_WORK=...
+```
+Or use `nexflow -e.NXF_WORK=...` approach.
+Ideally, should be overridable by the `-work-dir` (`-w`) option of `nextflow run`
+
+## Running a pipeline
+Once you have production (and nextflow) env ready, you can run pipelines.
+I.e.
+```
+CMD=<dba_alias>
+
+mkdir -p data
+pushd data
```

Comment:
should the `pushd` be removed from here as well?

Reply:
not from here